src/test/compile-fail/no-move-immovable-box-content.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(immovable_types)]
+
+use std::marker::Immovable;
+
+fn main() {
```

Comment:
Could you change this not to have an "use of possibly uninitialized variable" error, to prevent someone from refactoring this test away?

e.g., have a proper demonstration of unsoundness:
```Rust
fn move_from_box(b: Box<Immovable>) {
    *a; //~ ERROR cannot move immovable value out of a Box type
}

fn main() {
    let a = box Immovable;
    &*a; // ok
    move_from_box(a); // ok
}
```

Reply:
You can't actually create boxes with immovable types yet. I do not want to litter the code with `#[cfg(stage0)]`.

Reply:
Could you at least add a FIXME on the test?

Reply:
I changed the test to use your code.